content/pages/contact/speaker-bureau.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Have 18F speak at your event
+permalink: /speaker-bureau/
+layout: styled-container
+lead: At 18F, we’re proud to have a team of highly skilled and experienced professionals who are at the forefront of their respective fields. Many of our team members include global speakers and experts in their areas of practice.
+image: assets/img/speaking/ron-speaking-color-corrected.jpg
+image_alt_text: "18F Design Director Ron Bronson speaks to a crowd at AIGA Portland in 2024."
+email_call_to_action: true
+---
+
+If you’re looking for a speaker for your upcoming event or program, 18F staff have experience speaking about the following topic areas:
+
+- Engineering, including modernization, scaling, security, and Agile practices
+- Design, including UX design, user research, service design, information architecture, and content strategy
+- Product management in government
+- Stakeholder management
+- Government acquisition
+
+Formats including conference talks, workshops, and small group events.
```

Comment:
thing to consider: I know there's a call-to-action on the side, but, when I get to this end sentence, I'm finding myself wanting another sentence with a prose call to action, like a simple link.